region/norfair/east/Double Chamber.json

```diff
@@ -1171,6 +1174,40 @@
         {"heatFrames": 300}
       ]
     },
+    {
+      "link": [3, 4],
+      "name": "Ripper I-Frames, Spike Jump",
+      "requires": [
+        "canPrepareForNextRoom",
+        "canUseIFrames",
+        {"enemyDamage": {"enemy": "Ripper 2 (green)", "type": "contact", "hits": 1}},
+        {"or": [
+          "canHorizontalDamageBoost",
+          {"and": [
+            "HiJump",
+            "canTrickyJump"
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            "HiJump",
+            "canWalljump"
+          ]},
+          "canTrickySpringBallJump"
+        ]},
+        {"heatFrames": 300}
+      ],
+      "note": [
+        "Run off or perform a short hop off the ledge into the Ripper and damage boost to the left of the spikes.",
+        "Using the i-frames from the Ripper, quickly jump from the spikes to the item.",
+        "With HiJump, it is also possible to avoid the damage boost and land right of the crumble blocks and get a good jump to the left."
+      ],
+      "devNote": [
+        "FIXME: It is possible to avoid the Ripper and jump off of the crumble blocks.",
+        "With a Spring Ball jump, this requires an very precise jump and very quick pause to get as much horizontal speed as possible before the second jump.",
+        "With HiJump + Wall Jump, it is possible to crumble spin jump from the first crumbles to the second crumbles and then wall jump."
```

Comment:
This involves running on the crumbles while they break to gain speed.
I would do it by falling through the crumbles and jumping back up off the kago first so I could see where the blocks are.